lib/core/refreshController.js

```diff
@@ -1,5 +1,8 @@
 import ExpiredAuthSessionError from './includes/ExpiredAuthSessionError'
-
+/*
+* This class handles the refresh of tokens based on the expiry interval at 75%
+* so that we refresh before it expires. It will only refresh on Client side
+* */
```

Comment:
Actually, it handles the refresh of tokens based on token expiration. It checks token status in axios interceptor and auth middleware. 

Auto refresh was optional, but I just removed auto refresh function. It was useless. 😅

Reply:
I handle already a situation using 'laravel-echo' where the echo is disconnecting due to invalid access_token because it's not auto-refreshing. Made a workaround for this by myself but will be nice to have auto (timer) refresh options. Or it's already implemented but I'm doing smth wrong? :)